daemon/src/engine/framework/ConsoleHistory.h

```diff
@@ -31,18 +31,40 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef FRAMEWORK_CONSOLE_HISTORY_H_
 #define FRAMEWORK_CONSOLE_HISTORY_H_
 
+#include <string>
+#include <vector>
+#include <mutex>
```

Comment:
Well generally we have it set up so that headers don't need to include other headers. Since Common.h is directly or indirectly included in every .cpp file, just removing the #includes from headers should be fine.